frontend/src/components/UserProfile/UserProfile.js

```diff
@@ -119,9 +128,9 @@ function UserProfile(props) {
 
                 {/* Make following dynamic, based on user type will generate Patient Status/Medical License info */}
                 <div className={styles['internalCard']}>
-                    <div>Patient status:
+                    <div>Patient status: &nbsp;
```

Comment:
Is it normal that the patient status does not show?

Reply:
Patient status does not yet exist - for the time being, I will hard code some general text

Reply:
You can use the "userStatus" field present in the user documents from the database. You can also display what kind of user it is at the top of the screen (i.e. patient, doctor, immigration officer, etc.)